### PR TITLE
fix: update dependencies to remove vulnerabilities

### PR DESCRIPTION
**Pull request by murilohns**
update webpack and replace dependencies

replace was updated from 0.3.0 to 1.0.1 to remove a high vulnerability
dependency.

webpack-dev-server was updated from 1.14.1 to 3.1.14 to remove other vulnerability.
Because of this change, we had to install webpack-cli and update
webpack, node-sass and sass-loader

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -18,9 +18,13 @@ var baseConfig = {
     libraryTarget: 'var',
   },
   module: {
-    loaders: [
-      { test: /\.scss/, loaders: ["style-loader", "css-loader", "sass-loader"] },
-      { test: /\.json/, loader: "json-loader" },
```
[the remark this answers is not in the document]
Comment (murilohns) [reply webpack.config.js]: https://webpack.js.org/migrate/3/#json-loader-is-not-required-anymore

File: webpack.config.js [hunk 2]
```diff
@@ -18,9 +18,13 @@ var baseConfig = {
...
+    rules: [
+      {
+        test: /\.scss/,
+        use: [
+          "style-loader", "css-loader", "sass-loader"
+        ],
+      },
       { test: /\.coffee$/, loader: "coffee-loader" }
```
Comment (victorteodoro) [inline webpack.config.js]: Tem coffee script no componente??

Comment (murilohns) [reply webpack.config.js]: Sim senhor
https://github.com/pagarme/card/blob/master/src/coffee/card.coffee

Comment (victorteodoro) [reply webpack.config.js]: Boa! Esse é das antigas então :smile:

**Pull request merged**